theme/theme.inc

```diff
@@ -6,9 +6,9 @@
  */
 
 /**
- * Implements hook_preprocess().
+ * Implements template_preprocess_hook().
```

Comment:
Just minor. "hook" part should be upper case to denote that its the dymamic, module naming dependant part of the hook. Kinda picky so if you find it written differently in others module you can let if go